wallet-new/src/Cardano/Wallet/WalletLayer/Legacy.hs

```diff
@@ -94,6 +94,8 @@ bracketPassiveWallet =
         , _pwlCreateAddress         = pwlCreateAddress
         , _pwlGetAddresses          = pwlGetAddresses
 
+        , _pwlGetUtxos              = error "Method not implemented for legacy handler"
```

Comment:
I thought it was ^.^ ?

https://github.com/input-output-hk/cardano-sl/pull/3402/files#diff-b56e363cb80a182fee20d6f6dbccefe0R193

Reply:
At this level I am retrieving [(Account,Utxo)] for a given Wallet. Then ad the level of endpoint handler I will turn it into Utxo statistics. Why, maybe we are to extend endpoint and introduce some Account fields filtering, etc.